receiver/vcenterreceiver/documentation.md

```diff
@@ -325,6 +325,22 @@ The amount of memory that is ballooned due to virtualization.
 | ---- | ----------- | ---------- | ----------------------- | --------- |
 | By | Sum | Int | Cumulative | false |
 
+### vcenter.vm.memory.ssdswapped
+
+The amount of memory swapped to fast disk device such as SSD.
+
+| Unit | Metric Type | Value Type | Aggregation Temporality | Monotonic |
+| ---- | ----------- | ---------- | ----------------------- | --------- |
+| KiBy | Sum | Int | Cumulative | false |
+
+### vcenter.vm.memory.swapped
+
+The portion of memory that is granted to this VM from the host's swap space.
```

Comment:
the 'portion'  substituted by 'amount' which is aligned with other metric may be better?

Reply:
Sure, I can update that :) 
The description was taken from the VMware docs, but the two terms should be interchangeable.

Reply:
> Sure, I can update that :) The description was taken from the VMware docs, but the two terms should be interchangeable.

If it is from vmware docs, I think this can be kept as portion.